dlf/plugins/pageview/tx_dlf_utils.js

```diff
@@ -68,7 +68,7 @@ dlfUtils.createOl3Layers = function(imageSourceObjs, opt_origin) {
 
             tileSize = imageSourceObj.tilesize !== undefined && imageSourceObj.tilesize.length > 0
                 ? imageSourceObj.tilesize[0]
-                    : 256,
```

Comment:
Please put code beautifying always in a separate commit or even better in a separate pull request.